src/app/common-ui/Popup/style.scss

```diff
@@ -0,0 +1,104 @@
+.overlay {
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  top: 0px;
+  left: 0px;
+  right: 0px;
+  bottom: 0px;
+  background: rgba(0, 0, 0, 0.4);
+  z-index: 2;
+  mix-blend-mode: normal;
+}
+
+.container {
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  width: 100%;
+  height: 100%;
+  z-index: 3;
+
+  font-family: 'Mulish';
+  font-size: 16px;
+  line-height: 25px;
+
+  width: 600px;
+  margin: auto;
+  height: auto;
+  padding-bottom: 28px;
+
+  opacity: 1;
+  z-index: 3;
+  background: #ffffff;
+  box-shadow: 0px 0px 12px rgba(188, 188, 188, 0.25);
+  border-radius: 10px;
+}
+
+.titleBar {
+  width: 100%;
+  height: 56px;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.16);
+  align-items: center;
+  display: flex;
+}
+
+.titleContent {
+  font-family: 'Mulish';
+  width: 90%;
+  padding-left: 24px;
```

Comment:
```
  width: 90%;
  padding-left: 24px;
```

By using these two declarations, we cannot be sure if the div is centered. What if the container's width is changed?

Simply use `margin: 0 24px;` or `padding: 0 24px;`. We don't need to set `width` here.